samples/matter/common/src/bridge/ble_connectivity_manager.cpp

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "ble_connectivity_manager.h"
+
+#include <bluetooth/gatt_dm.h>
+#include <bluetooth/scan.h>
+#include <bluetooth/services/lbs.h>
+
+#include <platform/CHIPDeviceLayer.h>
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
+
+using namespace chip;
+
+BLEConnectivityManager BLEConnectivityManager::sBLEConnectivityManager;
+
+BT_SCAN_CB_INIT(scan_cb, BLEConnectivityManager::FilterMatch, NULL, NULL, NULL);
+
+BT_CONN_CB_DEFINE(conn_callbacks) = { .connected = BLEConnectivityManager::ConnectionHandler,
```

Comment:
I leave a note here, that if you define connection callbacks like here, they will be used also for Matter's BLE during the commissioning process. It isn't bad, but be aware of that.